egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+ . utils/parse_options.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+
+#################################################################
+#####             Downloading their git          ################
+#################################################################
+
+
+# Github AISHELL4 : https://github.com/felixfuyihui/AISHELL-4.git
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ] ; then 
+    FOLDER=git_aishell  
```

Comment:
Maybe you could put it along with `stage=0` so that user could easily found that and change the directory